webnn/resources/utils.js

```diff
@@ -328,6 +328,7 @@ const PrecisionMetrics = {
   elu: {ULP: {float32: 18, float16: 18}},
   expand: {ULP: {float32: 0, float16: 0}},
   gemm: {ULP: {float32: getGemmPrecisionTolerance, float16: getGemmPrecisionTolerance}},
+  instanceNormalization: {ATOL: {float32: 1/1024, float16: 1/512}},
```

Comment:
This's a placeholder. I tried to test by ULP with input [2, 3, 30, 30] and got the maximums ULP data is about 388. By this ATOL tolerance, tests could run pass. @fdwr Please help provide this tolerance criteria when you're available, thanks.

Reply:
The division by square root really complicates giving a straight forward answer (`(a - mean) * scale / sqrt(variance + epsilon) + bias`). Just based on the number of lossy operations, you'd expect a worst case of about...

```
6 + mean cost + variance cost
mean cost = ElementsToReduce + 1
variance cost = (Input.Sizes.W * Input.Sizes.H) * 3 + 1
```

...but because the subtraction can produce nearly equal numbers, and the division magnifies differences, we get a larger delta than that. Comfortable tolerant values for many GPU's would be ~840 ULP for float32 and 8'400 ULP for float16.

Reply:
@fdwr Thanks! 
I've updated tolerance as your recommended, please take another look, ☕